cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -24,9 +24,12 @@ $$ LANGUAGE plpgsql;
 
 CREATE TABLE IF NOT EXISTS demographics(
     census_geo_id varchar(255) NOT NULL,
+    census_block_name varchar(255),
```

Comment:
Added columns must be in a `ALTER TABLE` statement. As-is, these changes get skipped if someone already has a `demographics` table (we can't assume we'll just drop tables all the time in the future)

Reply:
As discussed offline, will leave as is for now since this is meant to handle the initial creation of the table / writing of the data. In the future I'll add alter table for any updates to avoid loading in all this data again. Thanks!